src/test/testing-library/AuthForm.test.tsx

```
@@ -260,4 +147,136 @@ describe('AuthForm', () => {
+
+    const { getAllByRole } = mockRenderAuthForm({
+      errorDetails: '',
+      notificationType: 'success',
+      mockOnSubmit,
+    });
+
+    const link: HTMLElement[] = getAllByRole('link');
+    expect(link[0]).toHaveAttribute('href', PRIVACY_POLICY_URL);
+  });
+  it('displays custom privacy policy URL when environment variable is set', async () => {
+    delete process.env.NEXT_PUBLIC_VILNACRM_PRIVACY_POLICY_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)